js/packages/cli/src/cli-nft.ts

```diff
@@ -66,18 +68,32 @@ programCommand('mint')
     if (toWallet) {
       receivingWallet = new PublicKey(toWallet);
     }
-    await mintNFT(
-      solConnection,
-      walletKeyPair,
-      url,
-      true,
-      collectionKey,
-      supply,
-      verifyCreators,
-      structuredUseMethod,
-      receivingWallet,
-    );
-  });
+    if (MintMultiple !== undefined) MintMultiple = 1;
```

Comment:
This condition is wired so if MintMultiple is not undefined you want to set it to 1 ? In this case, if will always be 1,,,

you probably want to have just default value on the variable and do not check it here at all

Reply:
because its changing it it wont even run since its set as a const.